src/components/Layout.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import LightricksLogo from '../assets/lightricksLogo.svg';
+import styles from './Layout.module.scss';
+
+export const Layout = ({ children, userImage }: { children: React.ReactNode, userImage?: string }) => (
+  <div className={styles.layout}>
```

Comment:
so in your implementation the layout div is scrollable. This is ok, but implies that the header is a part of the scrollable container.
So now let's make only the `app` div scrollable - you might not even need to make the header with `position: fixed`